tests/UtilTest.php

```diff
@@ -9,6 +9,7 @@
 use OpenApi\Annotations as OA;
 use OpenApi\Tests\Concerns\UsesExamples;
 use OpenApi\Util;
+use Symfony\Component\Filesystem\Path;
```

Comment:
This makes `symfony/filesystem` a direct dependency, so it should be added to `composer.json`

Reply:
After closer look, that would require php >= 8.0.2, so I have dropped this dependency and used simple lambda function instead.